libmproxy/proxy/config.py

```diff
@@ -133,10 +133,12 @@ def process_proxy_options(parser, options):
 
     if options.clientcerts:
         options.clientcerts = os.path.expanduser(options.clientcerts)
-        if not os.path.exists(options.clientcerts) or not os.path.isdir(options.clientcerts):
+        if not (os.path.exists(options.clientcerts) or
```

Comment:
> I was under the impression that using these function calls would help deter from special files being used

Yes, and then we get bug reports like #788 :stuck_out_tongue: . 
Just checking for `.exists` is fine I think, if it's not readable we'll raise an exception anyway. :smile: